blueprints/incredible-squaring/src/main.rs

```diff
@@ -25,22 +25,29 @@ use std::sync::Arc;
 
 #[async_trait::async_trait]
 trait GadgetRunner {
```

Comment:
@shekohex can we move these traits into the SDK?

Reply:
Looks like a good idea, will move it to the SDK.